.github/workflows/windows-amd64.yml

```diff
@@ -0,0 +1,655 @@
+name: "Windows amd64"
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+env:
+  TASKCLUSTER_TASK_DIR: ${{ github.workspace }}
+  TASKCLUSTER_ARTIFACTS: ${{ github.workspace }}/artifacts
+  TC_MSYS_VERSION: MSYS_NT-10.0-17763
+defaults:
+  run:
+    shell: bash
+jobs:
+  swig_Windows:
+    name: "Build SWIG for Windows"
+    runs-on: windows-2019
+    steps:
+      - name: Switch git-bash shell to MSYS2 shell by adding MSYS2 path to PATH front
+        run: echo "D:\a\_temp\msys\msys64\usr\bin" >> $GITHUB_PATH
```

Comment:
nit: can we have the value in an env variable? It would avoid having to hardcode it several times.